lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -179,6 +184,18 @@ def map_address(address)
         mapped
       end
 
+      def map_3ds(three_d_secure_options)
+        mapped = {
+          :eci => three_d_secure_options[:eci],
```

Comment:
The `xid` would be required in order for 3DS 1.x to be supported.

Reply:
Fixed now, remembered seeing this.